test/documentation/api-stability.js

```diff
@@ -0,0 +1,14 @@
+/* Enrico Seiler (enrico.seiler@fu-berlin.de), 2021 */
+
+$(document).ready(function()
+{
+    $('div.memitem').each(function(index) 
+    {
+        if ($(this).find('dl.experimental-api').length)
+            $(this).find('span.mlabels').prepend('<span class="mlabel experimental-api">experimental-api</span>');
+        else if ($(this).find('dl.stable-api').length)
+            $(this).find('span.mlabels').prepend('<span class="mlabel stable-api">stable-api</span>');
+        else
+            $(this).find('span.mlabels').prepend('<span class="mlabel no-api">no-api</span>');
+    });
+});
```

Comment:
Make sure it is no-api > experimental-api > stable-api in case multiple are there